src/full/Agda/TypeChecking/Reduce.hs

```diff
@@ -441,7 +441,8 @@ instance Reduce Sort where
         FunSort s1 s2 -> reduceB' (s1 , s2) >>= \case
           Blocked b (s1',s2') -> return $ Blocked b $ FunSort s1' s2'
           NotBlocked _ (s1',s2') -> do
-            case funSort' s1' s2' of
+            hasLevelUniv <- isLevelUniverseEnabled
+            case funSort' hasLevelUniv s1' s2' of
```

Comment:
I see many instances of the pattern
```haskell
hasLevelUniv <- isLevelUniverseEnabled
...  funSort(') hasLevelUniv s1 s2 ...
```
Would it make sense to have monadic `funSortM` and `funSortM'` that encapsulate this pattern?